src/components/inputs/number/number.test.tsx

```diff
@@ -3,6 +3,17 @@ import { act, render, screen, fireEvent } from '@testing-library/react';
 import { useFormProviderContext } from 'src/provider/form-provider';
 import NumberField from './number.component';
 
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key, options) => {
+      if (options && 'fieldDescription' in options) {
+        return `${options.fieldDescription} ${options.unitsAndRange || ''}`.trim();
```

Comment:
Since `extractFieldUnitsAndRange()` always returns a string:

```suggestion
        return `${options.fieldDescription} ${options.unitsAndRange}`.trim();
```